lib/routers/establishment/index.js

```diff
@@ -101,10 +75,58 @@ module.exports = (settings) => {
   router.use('/:establishment', permissions('establishment.read'));
 
   router.get('/:establishment', (req, res, next) => {
-    res.response = req.establishment;
-    next();
+    const { Establishment, PIL, Project } = req.models;
+
+    Promise.resolve()
+      .then(() => {
+        const query = Establishment.query()
+          .select('establishments.*')
+          .count('places.id', {as: 'placesCount'})
+          .leftJoin('places', 'places.establishment_id', 'establishments.id')
+          .findById(req.establishment.id)
+          .withGraphFetched('[authorisations]')
+          .groupBy('establishments.id');
+
+        const activePilsQuery = PIL.query()
+          .count('pils.id')
+          .where('status', 'active')
+          .andWhere('establishmentId', req.establishment.id)
+          .as('activePilsCount');
+
+        const activeProjectsQuery = Project.query()
+          .count('projects.id')
+          .where('status', 'active')
+          .andWhere('establishmentId', req.establishment.id)
+          .as('activeProjectsCount');
+
+        query.select(activePilsQuery, activeProjectsQuery);
+
+        return query;
+      })
+      .then(result => {
+        req.establishment = result;
+        res.response = req.establishment;
+      })
+      .then(() => next())
+      .catch(next);
+
   }, fetchOpenTasks(), fetchReminders('establishment'));
```

Comment:
I suspect given the frequency with which this endpoint gets hit that there's some benefit from splitting these out as well.